bool valid
int32 n_obs
int32 inliers
string dropped_faces
float32[] pos_err_m
float32[] ang_err_deg
